typegate/engine/Cargo.toml

```diff
@@ -3,6 +3,10 @@ name = "typegate_engine"
 edition.workspace = true
 version.workspace = true
 
+[features]
+default = []
+__runtime_js_sources = ["mt_deno/__runtime_js_sources"]
```

Comment:
any reason for prefixing with __ ?

Reply:
That's what the deno codebase for a similar flag and I decided against deviating from that. `mt_deno` does pas such a flag to `embedded-deno` when it's set as well.

Most likely, they added the underscores to indicate it's private.

No matter, I've removed the snapshots code.